manager.go

```diff
@@ -336,6 +336,16 @@ func (m *Manager) GetMap(name string) (*ebpf.Map, bool, error) {
 	return m.getMap(name)
 }
 
+// GetMaps - Return the list of eBPF maps in the manager
+func (m *Manager) GetMaps() (map[string]*ebpf.Map, error) {
+	m.stateLock.RLock()
+	defer m.stateLock.RUnlock()
+	if m.collection == nil || m.state < initialized {
+		return nil, ErrManagerNotInitialized
+	}
+	return m.collection.Maps, nil
```

Comment:
Should we create a copy here? This is dangerous, because you could accidentally modify internal state.

Reply:
I hesitated because we're giving pointers to the maps anyway ... But you're right, a copy would be better. Working on a PR now ! 👍🏻

Reply:
Fixed in https://github.com/DataDog/ebpf-manager/pull/105. Feel free to approve & merge if it works for you !